### ISSUE TITLE
(Suggestion) recherche par date

### ISSUE THREAD

**Issue by FLorianBlm** [WildCodeSchool/tours-php-2009-project2-mybooks#31]
La recherche par date devrait être sensiblement la même que la création de celle-ci.

**Comment by FLorianBlm** [comment]
Pour connaitre le bon format, notamment

**Comment by YannRaulet** [comment]
Effectivement, c'est un point intéressant, pour un livre sorti le 2013-06-07, nous pouvons  faire une recherche en tapant simplement l'année ou en tapant 2013-06-07.
